Automattic/vip-workflow-plugin
# Add notices for when post is locked from publishing

## Description

This PR adds a notice to the top of the editor when publish guard is enabled and the post is not in the last status.

## Steps to Test

1. Check out PR.
2. Enable publish guard in `VIP Workflow -> Notifications`
3. Create or edit a post that's not in `Pending review`
4. Verify that there is a notice that states the post is blocked from publishing 

![Screenshot 2024-07-15 at 4 29 41 PM](https://github.com/user-attachments/assets/556f6a5a-f138-408e-ab45-3200b7c4b2f6)
